Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM ubuntu:latest
 RUN apt-get update && apt-get install -y curl build-essential python2.7
 RUN curl -sL https://deb.nodesource.com/setup_6.x | bash -
-RUN apt-get update && apt-get -y install nodejs npm
+RUN apt-get update && apt-get -y install nodejs
```

Comment:
This is interesting as I was getting the error that `npm` was missing when I tried using this to build the zip file. Maybe a change in the packages for what `ubuntu:latest` was at the time.